docs/user/plagiarism-check.rst

```diff
@@ -172,6 +173,73 @@ The process is visualized in the following diagram.
+Subsequently, the student analyzes the feedback received and, if the submission period is still open, can submit a new solution.
+If the submission period has ended, the student can either dispute the plagiarism case or choose to take no action.
+
+The following activity diagram depicts the CPC flow described above.
+
+|cpc-activity-diagram|
+
+Instructors
+^^^^^^^^^^^
+
+Configuration of the CPC
+""""""""""""""""""""""""
+
+Instructors can configure various aspects of the CPC for each exercise. The following list describes all configuration options:
+
+* **Instructors can enable the CPC.** - Instructor can turn the CPC on and off for a particular exercise. Default: `OFF`.
+* **Instructors can enable the post due date checks.** - Instructor can configure the CPC to test the exercise one more time the night after the exercise due date. This allows to test submission posted shortly before the due date. Default: `OFF`.
+* **Instructors can set the response period for the significant similarity plagiarism case.** - Instructor can specify for how long after the exercise due date a student will be able to submit a complaint. Default: `7 days`. Min: `7 days`.
+* **Instructors can set the similarity threshold.** - Instructor can specify the similarity threshold for the plagiarism check performed by the CPC. Default: `90%`.
+* **Instructors can set the minimum score.** - Instructor can specify the minimum submission score for the plagiarism check performed by the CPC. Default: `0`.
+* **Instructors can set the minimum size.** - Instructor can specify the minimum submission size for the plagiarism check performed by the CPC. Default: `50`.
+
+Instructors can adjust the CPC configuration options on the exercise create and update page:
+
+|cpc-configuration|
+
+Post exercise due date steps and results verification
+"""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+After the exercise due date, instructors can manually review all of the significant similarity plagiarism cases and decide on their verdicts. Steps at this point are equivalent to the steps in manual checks flow.
+
+Instructors can check the plagiarism checks results CPC creates during and after the submission period using exactly the same steps and interface like with manual checks.
+
+Students
+^^^^^^^^
+
+When a student submits a work for which the CPC identifies significant similarity and creates a plagiarism case, the student receives a notification similar to what is sent when instructors create a classic plagiarism case.
+If the exercise due date has not passed, the student can submit updated work again.
+The CPC will check the submission the following night and will either remove or update the significant similarity plagiarism case based on the results of the check.
+Before the exercise due date, a student cannot see the submissions of other students involved in the significant similarity plagiarism case.
```

Comment:
Just for clarification: Afterwards they can?

Reply:
Yes